canonical-web-and-design/maas-ui
# feat(ui): events table

## Done

- Fetch and display the events in the logs tab.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the React logs tab for a machine.
- You should see a list of events (pagination etc. coming in future PRs).

## Fixes

Fixes: canonical-web-and-design/maas-squad#2217.
